src/store/store.ts

```diff
@@ -577,32 +582,40 @@ export const useStore = defineStore('store', {
       )
     },
 
-    /** Whether Amalgamation Information (regular) step is valid. */
-    isAmalgamationInformationRegValid (): boolean {
-      return (
-        this.getAmalgamatingBusinessesValid &&
-        !!this.getCorrectNameOption &&
-        this.getNameTranslationsValid
-      )
+    /** Whether Amalgamation Information step is valid. */
+    isAmalgamationInformationValid (): boolean {
+      if (this.isAmalgamationFilingRegular) {
+        return (
+          this.getAmalgamatingBusinessesValid &&
+          !!this.getCorrectNameOption &&
+          this.getNameTranslationsValid
+        )
+      } else {
+        return (
+          // *** FUTURE: verify/update this as needed
+          this.getAmalgamatingBusinessesValid
+        )
+      }
     },
 
-    /** Whether all the amalgamation (regular) steps are valid. */
-    isAmalgamationRegularValid (): boolean {
-      const isCertifyValid = this.getCertifyState.valid && !!this.getCertifyState.certifiedBy
-      const isCourtOrderValid = this.isRoleStaff ? this.getCourtOrderStep.valid : true
+    /** Whether all the amalgamation steps are valid. */
+    isAmalgamationValid (): boolean {
       const isFolioNumberValid = !this.isPremiumAccount || this.getFolioNumberValid
+      const isCourtOrderValid = this.isRoleStaff ? this.getCourtOrderStep.valid : true
+      const isCertifyValid = this.getCertifyState.valid && !!this.getCertifyState.certifiedBy
       const isStaffPaymentValid = this.isRoleStaff ? this.getStaffPaymentStep.valid : true
 
       return (
-        this.isAmalgamationInformationRegValid &&
+        this.isAmalgamationInformationValid &&
         this.isDefineCompanyValid &&
         this.isAddPeopleAndRolesValid &&
         this.isCreateShareStructureValid &&
-        isCertifyValid &&
-        isCourtOrderValid &&
+        this.getEffectiveDateTime.valid &&
```

Comment:
This is a bug-fix identified by Karim (thanks!). Now, invalid effective date-time blocks the filing from submission.